discovery-aws-api/src/main/resources/reference.conf

```diff
@@ -24,5 +24,7 @@ akka.discovery {
 
     class = akka.discovery.awsapi.ecs.EcsSimpleServiceDiscovery
 
+    cluster = "default"
```

Comment:
please document this key. When to set it, what;'s the use case for changing / having this setting

Reply:
is this value also default in ECS?

Reply:
It is for https://github.com/akka/akka-management/issues/180 I asume, please document this.
Could you please also explain this in documentation? Users should not be expected to read source details to find this feature

Reply:
Added something simple that I think should cover it. To add more would really just amount to duplicating AWS docs which users will need to have read anyway.